notebooks/python_intro/03-containers.md

```diff
@@ -108,17 +108,17 @@ Python reports an `IndexError` if we attempt to access a value that doesn’t ex
 <div class="alert alert-info">
 
 __Info__: 
-    
+
+* Slicing syntax: `L[start:stop:step]`
 * `L[start:stop]` contains the elements with indices i so `start <= i < stop`
 * i ranging from start to stop-1. Therefore, L[start:stop] has (stop-start) elements.
-* Slicing syntax: `L[start:stop:stride]`
 * All slicing parameters are optional
-* start and stop can be negative (counting backwards)
-   
+* All slicing parameters can be negative (counting backwards)
+
 </div> 
 
 
-When assigning a list to a new variable name, only a view is created and both names are linked to the same data in memory on your computer (cfr. two sticky notes specifiying the same). When changing one of them, the other will also be adjusted:
+When assigning a list to a new variable name, both names are linked to the same data in memory on your computer (cfr. two sticky notes specifiying the same). When changing one of them, the other will also be adjusted:
```

Comment:
This is the most significant edit that I made. The use of the term "view" is slightly incorrect or misleading here. It might be educationally sound to use for explaining pure python, but because the term "view" is then also used for numpy/pandas, this can become confusing and I think it's better to be correct here.

When doing `b = a`, the two objects are "identical" (point to the same thing in memory / have the same identity / `id(a) == id(b)`.